settings/movement.py

```diff
@@ -0,0 +1,169 @@
+from data.movement import MovementActions
+from memory.space import Allocate, Bank, Reserve, Read
+import instruction.asm as asm
+
+WALK_SPEED = 2
+SPRINT_SPEED = 3
+DASH_SPEED = 4
+
+class Movement:
+    def __init__(self):
+        import args
+        self.movement = args.movement or MovementActions.AUTO_SPRINT
+
+        if self.movement != MovementActions.ORIGINAL:
+            self.mod()
+
+    def mod(self):
+        length = 0
+        src = []
+
+        (length, src) = self.get_auto_sprint_src()
```

Comment:
Rather than calculate the length yourself, you should be able to just call 

space = Write(Bank.F0, src, "Sprint subroutine")